### ISSUE TITLE
Build packages for Alpine too

### ISSUE THREAD

**Issue by KOLANICH** [kaitai-io/kaitai_struct#514]

**Comment by GreyCat** [comment]
Probably possible, but, in general, it would be much easier for us to adopt this if Alpine packages would be supported in [sbt-native-packager upstream](https://github.com/sbt/sbt-native-packager/wiki/Tested-On). You might want to consider raising a feature request in their project?

**Comment by KOLANICH** [comment]
>You might want to consider raising a feature request in their project?

Yes, I may, but my feedback won't be valuable, I don't programm in Scala.

**Comment by GreyCat** [comment]
You can still do a feature request in sbt-native-packager project — it doesn't mean that you're supposed to provide a solution. There's actually some mentions of Alpine already:

https://github.com/sbt/sbt-native-packager/search?p=2&q=alpine&type=Issues

**Comment by KOLANICH** [comment]
https://github.com/sbt/sbt-native-packager/issues/1191
